test/ctia/flows/crud_test.clj

```diff
@@ -86,14 +86,15 @@
         "store-fn shall be applied to every entities")))
 
 (deftest create-events-test
-  (testing "flow-type :update should update the owner"
+  (testing "flow-type :update should preserve the entity owner"
     (let [updated-entities (atom [])
           flow-map         {:services        {:ConfigService {:get-in-config (constantly true)}}
-                            :create-event-fn (fn [entity _ _] (swap! updated-entities conj entity))
+                            :create-event-fn (fn [entity _ _ _] (swap! updated-entities conj entity))
                             :identity        (map->Identity {:login "test-user"})
                             :flow-type       :update
                             :entities        [{:one 1 :owner "Huey"}
                                               {:two 2 :owner "Dewey"}
                                               {:three 3 :owner "Louie"}]}]
      (#'flows.crud/create-events flow-map)
```

Comment:
I'd assert that the `(#'flows.crud/create-events flow-map)` call succeeded, otherwise the owners might not have changed b/c the update itself failed

Reply:
I finally radically addressed it, by tackling few points I was not comfortable with:
https://github.com/threatgrid/ctia/pull/1067/commits/700595a5b80b3418980c696c7ebccda621a3c866#diff-98019a373eff2f7053b805d6e606944309f8d42c562787f5749e809237a4f810R88